src/content/cloud/credentials.mdx

```diff
@@ -89,10 +89,14 @@ No resources found.
 
 ## Using your Kubernetes credentials
 
-Once you've downloaded your credentials, you'll have full access to your Kubernetes namespaces on Okteto directly from your terminal, using the tools you're already familiar with, such as `helm`, `kubectl`, `okteto` or `skaffold`.
+Before using your Kubernetes credentials, it is important to know that the kubeconfig file generated by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to authenticate you via Okteto to access your Kubernetes cluster. It is a common pattern used by many Kubernetes providers, such as Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), or Amazon Elastic Kubernetes Service (EKS) to connect to Kubernetes clusters in a more secure way. For this reason, it is important that the Okteto CLI is in your PATH and connected to your Okteto instance (having executed `okteto context use ...`).
+
+Once you've downloaded your credentials and it is configured, you'll have full access to your Kubernetes namespaces on Okteto directly from your terminal, using the tools you're already familiar with, such as `helm`, `kubectl`, `okteto` or `skaffold`.
 
 For example, you could use `kubectl` to deploy our `hello-world` application:
 
 ```terminal
 kubectl apply -f https://raw.githubusercontent.com/okteto/go-getting-started/master/k8s.yml
-```
\ No newline at end of file
+```
+
+> As mentioned before, credential plugins are a way of managing Kubernetes cluster credentials in a more secure way as you get time-bound tokens for your service account instead of using auto-generated secret-based ones. In case you face any issue with Okteto's credential plugin or you just simply want to keep using static auto-generated tokens, you can do it setting the environment variable `OKTETO_USE_STATIC_KUBETOKEN` to `true` before running any Okteto command. Be aware that using those static tokens are not recommended by Kubernetes and you will start getting warnings in your `kubectl` output from version `1.27` encouraging you to stop using them.
```

Comment:
```suggestion
> As mentioned before, credential plugins are a way of managing Kubernetes cluster credentials in a more secure way as you get time-bound tokens for your service account instead of using auto-generated secret-based ones. In case you face any issue with Okteto's credential plugin or you just want to keep using static auto-generated tokens, you can do it by setting the environment variable `OKTETO_USE_STATIC_KUBETOKEN` to `true` before running any Okteto command. Be aware that using those static tokens is not recommended by Kubernetes, and you will start getting warnings in your `kubectl` output from version `1.27` encouraging you to stop using them.
```